diagnostics-app/src/app/layout/side-nav/side-nav.data.ts

```diff
@@ -7,31 +7,23 @@
  * in the side-nav component.
  */
 
-export interface NavigationItem {
-  icon: string;
-  link: string;
+export interface NavigationButton {
```

Comment:
I think we should keep `NavigationItem` since it doesn't mean a clickable item here.

Reply:
SGTM. Thanks for the suggestion.